packages/react-table/src/components/Table/examples/Table.md

```diff
@@ -1947,11 +1823,1099 @@ class EditableRowsTable extends React.Component {
         aria-label="Editable Rows Table"
         variant={TableVariant.compact}
         cells={columns}
-        rows={rows}>
-          <TableHeader />
-          <TableBody />
+        rows={rows}
+      >
+        <TableHeader />
+        <TableBody />
       </Table>
     );
   }
 }
 ```
+
+## TableComposable examples
+
+### Composable: Basic
+
+A basic example using the composable table components. The `TableComposable` component differs from the regular `Table` component, in that it allows you to compose the table by nesting the relevant `Thead`, `Tbody`, `Tr`, `Th` and `Td` components within it. For a less declarative and more implicit approach, use the `Table` component instead.
+
+Some general notes:
+
+- Provide `dataLabel` prop to the `Td` components so that in mobile view the cell has a label to provide context.
```

Comment:
I love everything about this new API except for this fact because it prevents row logic from being separate from column logic. What do you think about some clever JS to pull strings from React nodes in `Thead` and then inject `dataLabel`s in `Tbody` `Td`s? Would this break the existing Table component?

Reply:
`dataLabel` passed to `Td` could be any string that the user wants to display in mobile view, typically it would be the column header. I am not opposed to your idea, I just want to be careful not to start going down the path of the current table component with too much context and state being passed around :) maybe this can be looked at as a further enhancement in the future?

Reply:
I agree I don't want to have to pass around this state. I've opened https://github.com/patternfly/patternfly/issues/3617